wd_capabilities.au3

```diff
@@ -145,93 +153,121 @@ EndFunc   ;==>_WD_CapabilitiesStartup
+		If Not @Compiled Then ConsoleWrite("! IFNC: TESTING NEW FEATURES #" & @ScriptLineNumber & $s_Parameters_Info & @CRLF)
 		Local $iResult = __WD_CapabilitiesInitialize($key, $value1)
 		If Not @error Then $_WD_CAPS__CURRENTIDX = $iResult
 		Return SetError(@error, @extended, $_WD_CAPS__CURRENTIDX)
 	EndIf
 	If $_WD_CAPS__CURRENTIDX = -1 Then Return SetError(1) ; must be properly initialized
 
-	#TODO use $value2 for "noProxy"  https://www.w3.org/TR/webdriver/#dfn-page-load-strategy
+;~ 	https://www.w3.org/TR/webdriver/#dfn-page-load-strategy
+;~ 	https://www.w3.org/TR/webdriver/#dfn-table-of-page-load-strategies
+
+	Local $s_SpecificOptions_KeyName = $_WD_CAPS__API[$_WD_CAPS__CURRENTIDX][$_WD_CAPS__SPECIFICVENDOR__ObjectName]
+	Local $v_WatchPoint
 	Local $s_Notation = ''
-	If $key = 'excludeSwitches' Then ; for adding "excludeSwitches" capability in specific/vendor capabilities : ........
-		$s_Notation = __WD_CapabilitiesNotation($_WD_CAPS__SPECIFICVENDOR__EXCSWITCH)
-	ElseIf $key = 'timeouts' Then ; for adding "proxy" capability in standard capability : https://www.w3.org/TR/webdriver/#capabilities
-		$s_Notation = __WD_CapabilitiesNotation($_WD_CAPS__STANDARD__TIMEOUTS)
-		$s_Notation &= '[' & $value1 & ']' ; here is specified keyName in {timeouts} JSON OBJECT
-		__WD_CapabilitiesSwitch($key, $value1, $value2)          ; as the notation was modified now parameters should be switched
-	ElseIf $key = 'proxy' Then ; for adding "proxy" capability in standard capabilities : https://www.w3.org/TR/webdriver/#dfn-proxy-configuration
-		$s_Notation = __WD_CapabilitiesNotation($_WD_CAPS__STANDARD__PROXY)
-		If $value1 = 'noProxy' Then ; for add string to "noProxy" JSON ARRAY in standard capabilities : https://www.w3.org/TR/webdriver/#dfn-proxy-configuration
-			$_WD_CAPS__API[$_WD_CAPS__CURRENTIDX][$_WD_CAPS__STANDARD__PROXY] += 1 ; default is -1 so first should be 0
-			Local $i_Current_noProxy = $_WD_CAPS__API[$_WD_CAPS__CURRENTIDX][$_WD_CAPS__STANDARD__PROXY]
-			$s_Notation &= '[noProxy][' & $i_Current_noProxy & ']' ; here is specified which one of [noProxy] JSON ARRAY element should be used
+
+
+	If StringRegExp($key, $_WD_CAPS_TYPES__STANDARD, $STR_REGEXPMATCH) Then ; for string/boolean value type in standard capability : https://www.w3.org/TR/webdriver/#capabilities
+		If $value2 <> '' Then
+			If Not @Compiled Then ConsoleWrite("! IFNC: TESTING NEW FEATURES #" & @ScriptLineNumber & $s_Parameters_Info & @CRLF)
+			If Not @Compiled Then MsgBox($MB_OK + $MB_TOPMOST + $MB_ICONERROR, "ERROR #" & @ScriptLineNumber, $s_Parameters_Info)
+			Return SetError($_WD_ERROR_NotSupported)
+		EndIf
+		$v_WatchPoint = @ScriptLineNumber
+		$s_Notation = __WD_CapabilitiesNotation($_WD_CAPS__STANDARD__CURRENT) & '[' & $key & ']'
+
+	ElseIf StringRegExp($key, $_WD_CAPS_TYPES__STANDARD_OBJECT, $STR_REGEXPMATCH) Then ; for string/boolean value type in standard capability : https://www.w3.org/TR/webdriver/#capabilities
```

Comment:
Comment says "string/boolean" but check is against standard object

Reply:
should be fixed soon in next commit

Reply:
[comments suplemented](https://github.com/Danp2/au3WebDriver/pull/263/commits/2dc23f0ede89e3f1770afa524acc856d0582f7d1)